pkg/blackboxexporter/blackboxexporter.go

```diff
@@ -0,0 +1,243 @@
+package blackboxexporter
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"github.com/openshift/route-monitor-operator/api/v1alpha1"
+	"github.com/openshift/route-monitor-operator/pkg/const/blackbox"
```

Comment:
maybe change the package name in the import as they are very similar by name but mean something else by context..

maybe constblackboxexporter?

Reply:
What do you think about moving them to the blackboxexporter package?

Reply:
did that, WDYT?